VenmoPayments/build.gradle

```diff
@@ -26,7 +25,7 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
     kotlinOptions {
-        jvmTarget = '1.8'
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
```

Comment:
With new modules should we consider targeting Java 11 or 17 instead? Or do we want to keep all modules aligned in Java version?

Reply:
I'd be down to go with 17 (or whatever BT v5 uses).

Reply:
Went for Java 11: [5a0abbd](https://github.com/paypal/paypal-android/pull/267/commits/5a0abbd2f5e14134797c15119f34adebdf0f0f59). Java 17 was giving the CI build problems for some reason.